src/StackExchange.Redis/StackExchange.Redis.csproj

```diff
@@ -10,6 +10,7 @@
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <DefineConstants Condition="'$(TargetFramework)' != 'net461'">$(DefineConstants);VECTOR_SAFE</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' != 'net461' and '$(TargetFramework)' != 'net472' and '$(TargetFramework)' != 'netstandard2.0'">$(DefineConstants);UNIX_SOCKET</DefineConstants>
```

Comment:
Recommend we define `NETCOREAPP3_1_OR_GREATER` here, which is landing in a future .NET 5.0 SDK (servicing release) we can remove: https://github.com/dotnet/designs/pull/164/files. This let's us cleanup nicely later.